_posts/0000-01-02-R-Andry.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Hello there, nice to meet you
```

Comment:
"I have wrestled with an alligator, done tussled with a whale, I done handcuffed lightning, throw thunder in jail !"

Reply:
"All of you chumps are gonna bow when I whoop him, I know you got him picked, but the man's in trouble, I'm gonna show you how great I am !"